Library/Homebrew/cmd/readall.rb

```diff
@@ -34,14 +38,23 @@ def readall_args
   def readall
     args = readall_args.parse
 
+    if args.no_simulate?
+      # odeprecated "--no-simulate", "nothing (i.e. not passing `--os` or `--arch`)"
+    end
```

Comment:
```suggestion
    # if args.no_simulate?
    #   odeprecated "--no-simulate", "nothing (i.e. not passing `--os` or `--arch`)"
    # end
```

Nit: Maybe it'd be better to just comment the whole thing out.